### PR TITLE
Paste

### PR DESCRIPTION
**Pull request by ellell**
Type: Minor
Review: @iefserge @kesla 

Support for pasting urls and parsing them to embeds.

### CODE REVIEW

File: example/client.js [hunk 1]
```diff
@@ -80,10 +80,10 @@ const items = [{
 }];
 
 const onInput = ({items}) => {
-  console.log('in client.js onInput');
-  app.mount(<Article items={items} onInput={onInput} />);
+  console.log('in client.js onInput or onBlur');
+  app.mount(<Article items={items} onInput={onInput} onBlur={onInput}/>);
```
Comment (kesla) [inline example/client.js]: nitpick: I like calling it `onChange` or something like that in these cases

File: lib/parse-url.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  if (!isUrl(content)) {
+    return null;
+  }
+
+  if (isImageUrl(content)) {
+    return {
+      type: 'embed',
+      embedType: 'image',
+      src: content
+    };
```
Comment (kesla) [inline lib/parse-url.js]: currently we host all images on our s3 - and this is something I foresee us doing in the future as well. This simply adds the image to the page - right?

Comment (kesla) [inline lib/parse-url.js]: also isImageUrl can make sync http calls - so a bit sceptical to that https://github.com/wzbg/is-image-url/blob/master/index.js#L22

File: example/client.js (hunk 1 above)
Comment (ellell) [inline example/client.js]: rename onBlur to onChange? or just the callback that's run on both?

Comment (kesla) [inline example/client.js]: callback.

**Comment by kesla** [comment]
one nitpick comment, rest LGTM 👯

**Pull request merged**